website/docs/d/cloudCredential.html.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: "rancher2"
+page_title: "Rancher2: rancher2_cloud_credential"
+sidebar_current: "docs-rancher2-datasource-cloud_credential"
+description: |-
+  Get information on a Rancher v2 Cloud Credentials.
+---
+
+# rancher2\_cloud\_credential
+
+Use this data source to retrieve information about a Rancher v2 setting.
```

Comment:
Not really a setting.

Rancher v2 cloud credential tied to a user. maybe?

Reply:
yep, it's a typo. Fixed.